sdk/communication/azure-communication-identity/pom.xml

```diff
@@ -63,7 +63,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.0.4</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->
```

Comment:
>>1.0.4 [](http://example.com/codeflow?start=14&length=6)

Follow up on why this changed? This should be 1.0.4 itself I think.

Reply:
It is because in version 1.1.1 it was incorrectly marked as 'current' and the previous commit reset it.

Reply:
yeah this change has to be picked up from master.
The Generate Patch PR might've not picked it up, as it creates the branch off the last release commit.